bob-common/src/configs/node.rs

```diff
@@ -492,6 +492,8 @@ pub struct Node {
     #[serde(default = "Node::default_http_api_address")]
     http_api_address: IpAddr,
     bind_to_ip_address: Option<SocketAddr>,
+    #[serde(default = "NodeConfig::default_filter_group_size")]
+    filter_group_size: usize,
```

Comment:
Please add comment about measurement unit of this value

Reply:
Renamed to `holder_group_size`. Now it should be cleaner